pkg/tfbridge/schema.go

```diff
@@ -324,6 +326,20 @@ func MakeTerraformInputs(
 	return inputs, cctx.Assets, err
 }
 
+func MakeTerraformInputs(
+	ctx context.Context, instance *PulumiResource, config resource.PropertyMap,
+	olds, news resource.PropertyMap, tfs shim.SchemaMap, ps map[string]*SchemaInfo,
+) (map[string]interface{}, AssetTable, error) {
+	return makeTerraformInputsHelper(ctx, instance, config, olds, news, tfs, ps, true)
+}
+
+func MakeTerraformInputsNoTFDefaults(
```

Comment:
Style nit. Looks a little peculiar - curious about this, since adding new functions takes up a new entry in the public Go doc for the package, WDYT about hiding it somehow. Could we make it `makeTerraforminputsNoTFDefaults` for now so it does not export? Could we make it an option on MakeTerraformInputs (adding functional options may be a non-breaking change).